cpptod.dd

```diff
@@ -829,6 +831,106 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(H3 <a name="functioninputparameters">Function Input Parameters</a>)
+
+$(H4 The C++ Way)
+
+	$(D C++ does not support explicit input parameters.) If you use a
+	"pass-by-value" parameter `void function(int inputOnlyImplicitly)` it can just
+	be an input parameter implicitly. If you use a "pass-by-reference" parameter
+	usually `const` is applied to the parameter
+	`void function(const int& inputOnlyParameterImplicitly)` that a forbidden
+	modification of the parameter in the function leads to a compilation error.
+
+$(CPPCODE
+void function(int implicitInputParameterPassedByValue) {
```

Comment:
The rest of the document uses a default function name, e.g. `foo` or anything you like.